lib/sarif/semgrep_sarif.rb

```diff
@@ -0,0 +1,121 @@
+require 'salus/bugsnag'
+
+module Sarif
+  class SemgrepSarif < BaseSarif
+    include Salus::SalusBugsnag
+
+    SEMGREP_URI = 'https://semgrep.dev/'.freeze
```

Comment:
Just realized we have SEMGREP_URI in here twice.  Safe to drop the first one here?